src/reputation/management/commands/initialize_reputation.py

```diff
@@ -14,7 +15,10 @@
 
 def calculate_user_score(user_id, algorithm_version, recalculate=False):
     user = User.objects.get(id=user_id)
-    author = user.author_profile
+    try:
+        author = user.author_profile
+    except ObjectDoesNotExist:
+        return "User does not have an author profile."
```

Comment:
This way, the function returns two different types, either a numeric type or a string. Handling of the exception could be done in the caller to retain a uniform return type.

Reply:
I will address this in a future PR